package.json

```diff
@@ -48,11 +48,12 @@
       }
     ],
     "okapiInterfaces": {
-      "inventory": "10.10",
+      "inventory": "10.10 11.0",
       "source-storage-records": "3.0",
-      "instance-storage": "7.0",
-      "holdings-storage": "3.0 4.4",
-      "item-storage": "8.0",
+      "search": "0.5",
+      "instance-storage": "7.0 8.0",
+      "holdings-storage": "3.0 4.4 5.0",
+      "item-storage": "8.0 9.0",
```

Comment:
Why does using the `search` interface need changes to the other interface versions as well?

Reply:
It doesn't; that was just an artifact of the merge with the main branch.

Reply:
I don't understand. If the main branch already expressed the interface dependencies that way, a comparison between this branch and the main branch should not describe it as a change.